client/lib/index.js

```diff
@@ -358,7 +358,7 @@ module.exports = class Client extends PassThrough {
 
 			const createWs = () =>
 				new Promise((resolve, reject) => {
-					const ws = new WebSocket(`ws://${this.uri.hostname}/start`);
+					const ws = new WebSocket(`ws://${this.uri.host}/start`);
```

Comment:
What's this change for?

Reply:
this change is needed in order to use the port number. the hostname attribute only includes the hostname without a port number